qml.qrc

```diff
@@ -280,7 +280,6 @@
         <file>atomic_defi_design/assets/images/coins/wbtc.png</file>
         <file>atomic_defi_design/assets/images/coins/okb.png</file>
         <file>atomic_defi_design/assets/images/coins/leo.png</file>
-        <file>atomic_defi_design/assets/images/coins/cfi.png</file>
```

Comment:
was it intended to delete some tokens logos in this commit?

Reply:
yes, those are the ones from the coins that were removed in a previous commit on https://github.com/KomodoPlatform/atomicdex-desktop-generics/commits/main... the coins are not in the config any more